clients/java/src/main/java/io/camunda/client/impl/CamundaClientImpl.java

```diff
@@ -1060,6 +1066,21 @@ public VariableGetRequest newVariableGetRequest(final long variableKey) {
     return new VariableGetRequestImpl(httpClient, variableKey);
   }
 
+  @Override
+  public VariableCreationRequestStep1 newVariableCreationRequest() {
+    return new CreateVariableImpl(httpClient, jsonMapper);
+  }
+
+  @Override
+  public VariableUpdateRequestStep1 newVariableUpdateRequest() {
+    return new UpdateVariableImpl(httpClient, jsonMapper);
+  }
+
+  @Override
+  public VariableDeleteRequestStep1 newVariableDeleteRequest() {
+    return new DeleteVariableImpl(httpClient);
+  }
```

Comment:
💭 For naming consistency, interactions with the engine are called `command`, while search (POST and GET by key) interactions are called `request`. This is neither written down anywhere nor enforced (because that is complex to do in the client). I believe we need to manifest this somewhere if we want this to be visible and adhered to.